### DISCUSSION TITLE
Add additional information to the user profile page

### DISCUSSION THREAD

**Discussion by ejektaflex** [modrinth/.github#9] [Feature requests]
**Is your feature request related to a problem? Please describe.**
The bio line is only a single line long right now - and doesn't have markdown support.

I was planning on putting something on the first line and a link on the third line, but that's not possible.

**Describe the solution you'd like**
A multiline textarea with markdown support would be great!

This may indirectly solve #284 , but an alternative solution would also to add fields for socials like:
* Website (if you have a personal website)
* GitHub username (auto turns into GH url?)
* Discord handle

**Comment by Rafii2198** [comment]
If fields were to be added then adding Mastodon user page verification would be really easy to do. Just a rel="me" with the link provided in Mastodon field as href.

Overall fields have this advantage of being consistent across the pages, they would fit pretty clean among the user page same as they fit really well the project page.

**Comment by Minenash** [comment]
This should apply to orgs as well
